.github/workflows/ci.yaml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  ci:
+    name: Build + Test
+    runs-on: macOS-latest # enables hardware acceleration in the virtual machine
+    strategy:
+      matrix:
+        api-level: [21, 29]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Make files executable
+        run: chmod +x ./JetNews/gradlew && chmod +x ./JetNews/checksum.sh
+
+      - name: Generate cache key
+        run: ./JetNews/checksum.sh checksum.txt
+
+      - name: Copy CI gradle.properties
+        run: mkdir -p ~/.gradle ; cp .github/ci-gradle.properties ~/.gradle/gradle.properties
+
+      - uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches/modules-2
+          key: ${{ runner.os }}-gradlemodules-${{ hashFiles('checksum.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-gradlemodules-
+
+      - uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches/jars-3
+          key: ${{ runner.os }}-gradlejars-${{ hashFiles('checksum.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-gradlejars-
+
+      - uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches/build-cache-1
+          key: ${{ runner.os }}-gradlebuildcache-${{ hashFiles('checksum.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-gradlebuildcache-
+
+      - name: Ktlint
+        run: cd JetNews && ./gradlew ktlint --stacktrace
```

Comment:
Could this not be:

`./gradlew JetNews:ktlint`?

Reply:
Tried this before and now again (because maybe I was doing something wrong before) but still getting the following error: `Project 'JetNews' not found in root project 'compose-samples'`